sonarqube/helm/sonarqube/templates/postgres-secrets.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sonarqube.plural-postgres-sonarqube.credentials.postgresql.acid.zalan.do
+  labels:
+{{ include "sonarqube-plural.labels" . | indent 4 }}
+stringData:
+  username: {{ .Values.postgres.user }}
```

Comment:
You don't actually need to generate this, the operator will do it automatically for you (it's only generated as a manual workaround when upstream charts require us to pass the password or full dsn manually)